src/Catalyst.Node.POA.CE.Tests/IntegrationTests/PoaConsensusTests.cs

```diff
@@ -93,29 +103,40 @@ public async Task Run_ConsensusAsync()
 
             await Task.Delay(Debugger.IsAttached
                     ? TimeSpan.FromHours(3)
-                    : CycleConfiguration.Default.CycleDuration.Multiply(1.3))
+                    : CycleConfiguration.Default.CycleDuration.Multiply(2.3))
                .ConfigureAwait(false);
 
-            var dfsDir = Path.Combine(FileSystem.GetCatalystDataDir().FullName, "dfs");
-            Directory.GetFiles(dfsDir).Length.Should().Be(1,
-                "only the elected producer should score high enough to see his block elected.");
+            //At least one delta should be produced
+            var maxDeltasProduced = 1;
+            var files = new List<string>();
+            for (var i = 0; i < _nodes.Count; i++)
+            {
+                var dfsDir = Path.Combine(FileSystem.GetCatalystDataDir().FullName, $"producer{i}/dfs", "blocks");
+                var deltaFiles = Directory.GetFiles(dfsDir).Select(x => new FileInfo(x).Name).ToList();
+                maxDeltasProduced = Math.Max(maxDeltasProduced, deltaFiles.Count());
+                files.AddRange(deltaFiles);
+            }
+
+            files.Distinct().Count().Should().Be(maxDeltasProduced,
+                "only the elected producer should score high enough to see his block elected. Found: " +
+                files.Aggregate((x, y) => x + "," + y));
 
             _endOfTestCancellationSource.CancelAfter(TimeSpan.FromMinutes(3));
         }
 
-        protected override void Dispose(bool disposing)
-        {
-            base.Dispose(disposing);
-            if (!disposing) return;
+        //protected override void Dispose(bool disposing)
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)